docs/4_the-formal-stuff/Career Development/readme.md

```diff
@@ -40,12 +40,12 @@ For the Engineering verticals the 5th axis is:
 And for the Research the 5th axis is:
```

Comment:
The above should say that `For the Engineering verticals the 5th axis is:` `System` and not `Technical expertise`